pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter_test.go

```diff
@@ -483,13 +483,15 @@ var _ = Describe("Validating VMICreate Admitter", func() {
 	Context("with VirtualMachineInstance spec", func() {
 		It("should accept valid machine type", func() {
 			vmi := api.NewMinimalVMI("testvmi")
-			if webhooks.IsPPC64(&vmi.Spec) {
+
+			switch vmi.Spec.Architecture {
```

Comment:
This test depends on the arch of the host, right? Can you make it arch independent? I.e. converting it to table?

Reply:
Will do

Reply:
Having looked at the code, it seems the vmi Architecture field was never actually set, so the test just checked amd64 anyway, making the whole if/switch unnecessary.
I have converted it to a Table and fixed the test such that different arches are actually tested.

Reply:
I have also found out that the validation of the machine type is not quite working as expected, since everything which contains `q35` seems to be valid, including `reiterithq35`. I will check that out further and open an issue if required.